### DISCUSSION TITLE
Recursive tasks disappear from Thunderbird!

### DISCUSSION THREAD

**Discussion by luckehall** [kewisch/gdata-provider#626] [Support]
Since the "Provider for Google Calendar" last update, Thunderbird no longer allows recursive tasks!

**Comment by kewisch** [comment]
Do you mean recurring tasks? I don't believe these are supported at all by the Google Tasks API, so if it worked before maybe there was a bug hiding the UI for it?

**Reply by clari7744** [reply]
Are you saying that recurring tasks can't be fetched when using the Google API at all? It only returns one-time tasks when fetching the list of tasks associated with an account?

**Reply by kewisch** [reply]
Google does not expose repeating info for the tasks, so I can't make them repeating: https://developers.google.com/tasks/reference/rest/v1/tasks#resource
